docs/reference/experimental/async/submission.md

```diff
@@ -0,0 +1,19 @@
+# Submission
+
+Contained within this file are experimental interfaces for working with the Synapse Python
+Client. Unless otherwise noted these interfaces are subject to change at any time. Use
+at your own risk.
+
+## API Reference
+
+::: synapseclient.models.Submission
+    options:
+        inherited_members: true
```

Comment:
With `inherited_members: true` we will still need to manually add in the things the class in inheriting from in the `members` section below. So in this case I think we would add all of this:

```
            - get_acl_async
            - get_permissions_async
            - set_permissions_async
            - delete_permissions_async
            - list_acl_async
```